packages/terra-date-picker/tests/jest/__snapshots__/DatePicker.test.jsx.snap

```diff
@@ -102,8 +102,8 @@ exports[`correctly applies the theme context className 1`] = `
                 }
                 isIncomplete={false}
                 isInvalid={false}
-                maxDate={"2017-01-01T00:00:00.000Z"}
-                minDate={"2017-01-01T00:00:00.000Z"}
+                maxDate="2100-12-31"
+                minDate="1900-01-01"
```

Comment:
DateInput expects strings so DatePicker was updated to use the string values of max/minDates instead of the moment objects returned from DateUtil which was returning the mocked value before